flake8_trio/visitors/flake8triovisitor.py

```diff
@@ -47,8 +46,6 @@ def __init__(self, shared_state: SharedState):
             "typed_calls",
         }
 
-        self.suppress_errors = False
-
```

Comment:
no longer used since 91X doesn't visit loop bodies twice, where it previously needed to suppress errors during one of the passes to not duplicate errors.